docs/container.md

```diff
@@ -0,0 +1,65 @@
+# Container Instrumentation
+
+This document details the steps required to instrument a Java based container to use the opentracing [specialagent](https://github.com/opentracing-contrib/java-specialagent).
```

Comment:
You can instrument a Java-based container to use the [OpenTracing Special Agent](https://github.com/opentracing-contrib/java-specialagent) by following the steps on this page. 
OR:
To instrument.... follow the steps...